apps/artemis-web/components/hijack-info/hijack-info.tsx

```diff
@@ -104,7 +103,208 @@ class HijackInfoComponent extends Component<any, any> {
+                        ? exportHijack(hijackKey, this.props._csrf)
+                        : sendHijackData(e, {
+                          hijackKey: hijackKey,
+                          selectState: mRef.current.value,
+                          prefix: this.props.hijackDataState.prefix,
+                          hijack_as: this.props.hijackDataState.hijack_as,
+                          type: this.props.hijackDataState.type,
+                          _csrf: this.props._csrf,
+                        })
+                  }
+                  style={{ marginRight: '5px' }}
+                  id="apply_selected"
+                  type="button"
+                  className="btn btn-primary btn-md"
+                >
+                  Apply
+                </button>
+              </div>
+            </div>
+          </div>
+        </div>
+        <div className="row">
+          <div className="col-lg-12">
+            <div className="card">
+              <div className="card-header">
+                Comments{' '}
+                <button
+                  style={{ marginRight: '5px', float: 'right' }}
+                  id="edit_comment"
+                  type="button"
+                  className={`btn btn-${!this.state.editComment ? 'primary' : 'secondary'
+                    } btn-md`}
+                  onClick={(e) => {
+                    if (this.state.editComment)
+                      this.setState({
+                        commentSuccess: submitComment(e, {
+                          commentRef,
+                          hijackKey,
+                          _csrf: this.props._csrf,
+                        }),
+                      });
+                    else commentRef.current.focus();
+
+                    this.setState({
+                      editComment: !this.state.editComment,
+                    });
+                  }}
+                >
+                  {!this.state.editComment ? 'Edit' : 'Save'}
+                </button>
+              </div>
+              <div className="card-body">
+                <h1
+                  style={{
+                    display: this.state.commentSuccess ? 'none' : 'block',
+                    color: 'red',
+                  }}
+                >
+                  {' '}
+                  Config failed to update.{' '}
+                </h1>
+                <Editor
+                  ref={commentRef}
+                  readOnly={!this.state.editComment}
+                  editorState={this.props.editorState}
+                  onChange={this.props.setEditorState}
+                />
+              </div>
+            </div>
+          </div>
+        </div>
+
+        <div className="row">
+          <div className="col-lg-12">
+            <div className="card" style={{ marginTop: '12px', visibility: this.state.gripState ? "visible" : "hidden" }}>
+              <div className="card-header">
+                CAIDA GRIP{' '}
```

Comment:
could you rename to `Also detected by the GRIP project` (minor)? Instead of CAIDA GRIP.

Reply:
Also please do one more renaming because somehow it slipped through the cracks: in the hijack info page, it should be "Hijacker AS" not "Hijacked AS" (important)